pkg/flowkit/account.go

```diff
@@ -111,6 +111,45 @@ func toConfig(account Account) config.Account {
 	}
 }
 
+func AccountFromFlow(account *flow.Account, name string, key crypto.PrivateKey) (*Account, error) {
+	flowkitAccount := &Account{
+		name:    name,
+		address: account.Address,
+	}
+
+	if len(account.Keys) > 0 {
```

Comment:
I think that both the `AccountFromFlow` and `AccountFromFlowEncrypted` functions should be more strict and return an error if `len(account.Keys) != 1`. 

This prevents an exploit where an attacker front-runs account creation transactions and submits their own account creation transaction with the victim's public key, as well as their own additional malicious key. A user could then be tricked into sending funds to an account address that they only partially own.

Reply:
In addition to that, we should probably also throw an error if `defaultKey.Weight != 1000`. This is an unlikely edge case, but an account is unusable if it has only one key and that key has less than 1000 weight.